translate/cloud-client/snippets.py

```diff
@@ -23,7 +25,9 @@
 
 import argparse
 
+
 from google.cloud import translate
+from six import python_2_unicode_compatible
```

Comment:
Oh no, not this. If `text` comes from the command line you should just be able to do `text.encode('utf-8')`. You' won't even have to reference the specific type (bytes/str).